packages/wrangler/src/versions/deploy.ts

```diff
@@ -205,6 +209,61 @@ function getConfig(
 	return config;
 }
 
+/**
+ * Prompts the user for confirmation when overwriting the latest deployment, given that it's split.
+ */
+export async function confirmLatestDeploymentOverwrite(
+	accountId: string,
+	scriptName: string
+) {
+	try {
+		const latest = await fetchLatestDeployment(accountId, scriptName);
+		if (latest && latest.versions.length >= 2) {
+			// Fetch the version details.
+
+			const versionCache: VersionCache = new Map();
+			const versionIds = latest.versions.map((v) => v.version_id);
+			await fetchVersions(accountId, scriptName, versionCache, ...versionIds);
+
+			const versions = latest.versions.map((v) => {
+				const version = versionCache.get(v.version_id);
+				assert(version);
+				return version;
+			});
+
+			const traffic = new Map(
+				latest.versions.map((v) => [v.version_id, v.percentage])
+			);
+
+			// Print message and confirmation.
+
+			cli.warn(
+				`Your last deployment has multiple versions. To progress this deployment use "wrangler versions deploy" instead.`,
```

Comment:
copy nit: "that" instead of "this" because "this" deployment should refer to one being triggered by this command whereas "that" deployment would refer to the "last" deployment mentioned in this line?

cc @tanushree-sharma

```suggestion
				`Your last deployment has multiple versions. To progress that deployment use "wrangler versions deploy" instead.`,
```